integration/fixtures/regenerate-certs.sh

```diff
@@ -10,6 +10,9 @@ rm -rf out
 certstrap init --common-name "ca" --passphrase ""
 certstrap request-cert --common-name "server" --domain "localhost" --ip "127.0.0.1" --passphrase ""
 certstrap sign server --CA "ca"
+certstrap init --common-name "ca-bad" --passphrase ""
+certstrap request-cert --common-name "server-bad" --passphrase ""
+certstrap sign server-bad --CA "ca-bad"
```

Comment:
Why save `ca-bad.crl` and `ca-bad.crt` when we don't actually use them in tests?

Reply:
Fair. I needed them i think to be on a different ca, but I'll check again.

Reply:
didn't need a different ca. idk how i was setting up the test wrong before.